feature_creation_scripts/antarctic_ocean_regions/antarctic_ocean_regions.py

```diff
@@ -211,40 +238,73 @@ def split_rectangle(lon0, lon1, lat0, lat1, name, author, tags, fcContour):
     return fc
 
 
+def split_polygon(lon_vector, lat_vector, name, author, tags, fcContour):
+    fc = make_polygon(lon_vector, lat_vector, name, author, tags)
+
+    fcDeep = fc.difference(fcContour)
+
+    props = fcDeep.features[0]['properties']
+    props['name'] = props['name'] + ' Deep'
+    props['tags'] = props['tags'] + ';Deep'
+    props['zmin'] = -1000.
+    props['zmax'] = -400.
+
+    fcShelf = fc.difference(fcDeep)
+
+    props = fcShelf.features[0]['properties']
+    props['name'] = props['name'] + ' Shelf'
+    props['tags'] = props['tags'] + ';Shelf'
+    props['zmin'] = -1000.
+    props['zmax'] = -200.
+
+    fc.merge(fcDeep)
+    fc.merge(fcShelf)
+
+    return fc
+
+
 def main():
-    author = 'Xylar Asay-Davis'
+    xylar = 'Xylar Asay-Davis'
+    carolyn = 'Carolyn Begeman'
     timTags = 'Antarctic;Timmermann'
     orsiTags = 'Antarctic;Orsi'
+    antTag = 'Antarctic'
 
     # make a geometric fieatures object that knows about the geometric data
     # cache up a couple of directories
     gf = GeometricFeatures('../../geometric_data')
 
     bedmap2_bin_to_netcdf('bedmap2.nc')
 
-    fcContour700 = get_longest_contour(contourValue=-700., author=author)
-    fcContour800 = get_longest_contour(contourValue=-800., author=author)
+    fcContour700 = get_longest_contour(contourValue=-700., author=xylar)
+    fcContour800 = get_longest_contour(contourValue=-800., author=xylar)
 
     fc = FeatureCollection()
 
-    fcWeddell = split_rectangle(
-        lon0=-63., lon1=0., lat0=-80., lat1=-65., name='Weddell Sea',
-        author=author, tags=timTags, fcContour=fcContour800)
+    fcWeddell = split_polygon(
+        lon_vector=[-100., -86., -63., -66., -60., -51., 0., 0.],
+        lat_vector=[-81., -75., -73., -67., -64., -62., -62., -90.],
```

Comment:
Does it still make sense to have a separate feature for the full Weddell?  Or can we just combine the the Eastern and Western (as we do below) and then split them into shelf and deep?  We had a separate definition for the full Weddell before because we wanted to have the shelf/deep definitions match the Timmermann and Helmer paper but that doesn't apply anymore with these changes.

Reply:
I think I would prefer to remove the separate definition of the Weddell Sea and just split the combined Eastern and Western.  I will do that as soon as we agree on the definition of the Eastern below.

Reply:
Were we using the Timmermann and Helmer paper for comparison? If not, and also considering that the new regions depart from the definition in that paper, I agree that we could just keep one Weddell.

I guess I am a little confused about why Caroline's modifications still had a discontinuity, but I hope the latest changes work.

Reply:
> Were we using the Timmermann and Helmer paper for comparison? 

Yes, we were at some point but we can't do that either way at this point.  We have modified all 3 Weddell regions and the Bellighshausen region compared with that paper.  I don't think we're that keen to keep comparing with them.

Reply:
Great. I can commit this change soonish.